### ISSUE TITLE
st+ working for rgui but failing for rstudio and sublime repl

### ISSUE THREAD

**Issue by ghonk** [randy3k/SendTextPlus#34]
Hello, I don't expect you'll be able to solve this but I was hoping that I could just inquire about some of the possibilities for why st+ would work for rgui but fail for rstudio. Sending to sublimeREPL has never worked for me, but Rstudio used to work.

here's my session info from RStudio
```
sessionInfo()
R version 3.2.3 (2015-12-10)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows >= 8 x64 (build 9200)

locale:
[1] LC_COLLATE=English_United States.1252  LC_CTYPE=English_United States.1252    LC_MONETARY=English_United States.1252 LC_NUMERIC=C                           LC_TIME=English_United States.1252    

attached base packages:
[1] grid      stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] ggplot2_2.0.0     tikzDevice_0.10-1

loaded via a namespace (and not attached):
 [1] colorspace_1.2-6   scales_0.3.0       magrittr_1.5       plyr_1.8.3         tools_3.2.3        RColorBrewer_1.1-2 gtable_0.1.2       reshape2_1.4.1     Rcpp_0.12.4        stringi_1.0-1     
[11] filehash_2.3       digest_0.6.8       stringr_1.0.0      munsell_0.4.2    
```

**Comment by randy3k** [comment]
How does it not work? Does it send something at all?

**Comment by ghonk** [comment]
nope nothing at all, the cursor hourglass appears for a brief moment, also tried adding RStudio to the path

**Comment by randy3k** [comment]
You have to open a RStduio Windows first in order to use RStudio.

**Comment by ghonk** [comment]
Ok, thanks for the advice. That wasn't my issue. However, I just found out that running sublime as admin addresses the issue. Thanks for your help and work on the package, happy to consider ithis resolved unless you have any advice about possible causes for the issue below.

For sublimeREPL, when I open Sublime and try to send to REPL it fails with the error below. When I open the REPL, there is seemingly no reaction to ctrl+enter

![untitled picture](https://cloud.githubusercontent.com/assets/3682565/15194167/832c733a-178f-11e6-9a5e-8db284526906.png)

**Comment by randy3k** [comment]
Did you launch a R REPL?

Actually, I don't recommend using SubliemREPL. 
1) it is not actively maintained. 
2) The embedded R is invisible to users, if the R kernel dies, SublimeREPL does know how to handle it.

**Comment by ghonk** [comment]
Yes I did, after initialization there is no visible reaction to ctrl+enter. Just weird that I get the "Cannot find" message when it's not open and then no action when it is open.

I get your points about REPL, sending commands to it with st+ has never worked for me, so no big loss there.

**Comment by randy3k** [comment]
FYI,

On my Windows 10, I would send to RStudio by opening Sublime Text with admin rights. May be it is the way that your Sublime Text was installed causing the problem?
